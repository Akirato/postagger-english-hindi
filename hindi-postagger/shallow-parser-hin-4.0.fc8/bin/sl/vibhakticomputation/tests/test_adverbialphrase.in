<Sentence id="1">
1	((	NP	<fs af='rAma,n,m,sg,3,d,0,0' head="rAma">
1.1	rAma	NNP	<fs af='rAma,n,m,sg,3,d,0,0' name="rAma">
	))		
2	((	RBP	<fs af='hameSA,adv,,,,,,' head="hameSA">
2.1	hameSA	RB	<fs af='hameSA,adv,,,,,,' name="hameSA">
2.2	se	PSP	<fs af='se,psp,,,,,,'>
2.3	hI	RP	<fs af='hI,adv,,,,,,' poslcat="NM">
	))		
3	((	VGF	<fs af='kara,v,m,sg,any,,wA,wA' head="karawA">
3.1	karawA	VM	<fs af='kara,v,m,sg,any,,wA,wA' name="karawA">
3.2	AyA	VAUX	<fs af='A,v,m,sg,any,,yA,yA'>
3.3	hE	VAUX	<fs af='hE,v,any,sg,2,,hE,hE'>
	))		
</Sentence>
