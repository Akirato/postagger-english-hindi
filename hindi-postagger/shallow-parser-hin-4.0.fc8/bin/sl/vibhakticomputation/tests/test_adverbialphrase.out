<Sentence id="1">
1	((	NP	<fs af='rAma,n,m,sg,3,d,0,0' head="rAma">
1.1	rAma	NNP	<fs af='rAma,n,m,sg,3,d,0,0' name="rAma">
	))		
2	((	RBP	<fs af='hameSA,adv,,,,,0_se,' head="hameSA" vpos="vib1_2_RP">
2.1	hameSA	RB	<fs af='hameSA,adv,,,,,,' name="hameSA">
2.2	hI	RP	<fs af='hI,adv,,,,,,' poslcat="NM">
	))		
3	((	VGF	<fs af='kara,v,m,sg,2,,wA_A+yA_hE,wA' head="karawA" vpos="tam1_2_3">
3.1	karawA	VM	<fs af='kara,v,m,sg,any,,wA,wA' name="karawA">
	))		
</Sentence>

